src/main/java/com/lorepo/icplayer/client/model/page/PageList.java

```diff
@@ -260,36 +261,40 @@ else if(node instanceof PageList){
 	@Override
 	public void load(Element rootElement, String url) {
 		//for IXMLSerializable interface
-		load(rootElement, url, null, 0);
+		load(rootElement, url, null, 0, null);
 	}
 
 
-	public int load(Element rootElement, String url, ArrayList<Integer> subsetOfPages, int pageIndex) {
+	public int load(Element rootElement, String url, ArrayList<Integer> subsetOfPages, int pageIndex, ArrayList<Integer> pagesMapping) {
```

Comment:
Nie byłoby lepiej to przeciążyć, żeby nie trzeba było ustawiać ostatniego argumentu na null?